test/utils/helpers.test.js

```diff
@@ -29,3 +29,80 @@ describe('Helpers', () => {
         assert.ok(!isMinimumServerVersion());
     });
 });
+
+describe('Utils.isEmail', function() {
+    it('', function() {
+        for (const data of [
+            {
+                email: 'prettyandsimple@example.com',
+                valid: true,
+            },
+            {
+                email: 'very.common@example.com',
+                valid: true,
+            },
+            {
+                email: 'disposable.style.email.with+symbol@example.com',
+                valid: true,
+            },
+            {
+                email: 'other.email-with-dash@example.com',
+                valid: true,
+            },
+            {
+                email: 'fully-qualified-domain@example.com',
+                valid: true,
+            },
+            {
+                email: 'user.name+tag+sorting@example.com',
+                valid: true,
+            },
+            {
+                email: 'x@example.com',
+                valid: true,
+            },
+            {
+                email: 'example-indeed@strange-example.com',
+                valid: true,
+            },
+            {
+                email: 'admin@mailserver1',
+                valid: true,
+            },
+            {
+                email: '#!$%&\'*+-/=?^_`{}|~@example.org',
+                valid: true,
+            },
+            {
+                email: 'example@s.solutions',
+                valid: true,
+            },
+            {
+                email: 'Abc.example.com',
+                valid: false,
+            },
+            {
+                email: 'A@b@c@example.com',
+                valid: false,
+            },
+            {
+                email: '<Jonathan Fritz> jonathan.fritz@mattermost.com',
```

Comment:
Can we just replace this with other fictional email?

Reply:
Yup, of course.